api/v1beta1/activemqartemis_types.go

```diff
@@ -685,6 +685,7 @@ const (
 	DeployedConditionNotReadyReason         = "PodsNotReady"
 	DeployedConditionZeroSizeReason         = "ZeroSizeDeployment"
 	DeployedConditionValidationFailedReason = "ValidationFailed"
+	DeployedConditionResourceErrorReason    = "ResourceError"
```

Comment:
would ResourceCRUDerror be better?

Reply:
I vote for `ResourceError` or `DeploymentResourceError`

Reply:
now thinking: KindCRUDError or is that more incomprehensible? it is more accurate

Reply:
resource means cpu/memory etc in kube, a type is a kind, and we are doing CRUD as part of the reconcile loop.

Reply:
hmm, the crud acronym is too much, going to revert to the original, resource is more general than CPU and RAM and is suitable.